src/main/java/io/managed/services/test/Environment.java

```diff
@@ -111,23 +109,21 @@ public class Environment {
     public static final Path LOG_DIR = getOrDefault(LOG_DIR_ENV, Paths::get, Paths.get(SUITE_ROOT, "target", "logs")).resolve("test-run-" + DATE_FORMAT.format(LocalDateTime.now()));
 
     // sso.redhat.com primary user (See README.md)
+    public static final String PRIMARY_OFFLINE_TOKEN = getOrDefault(PRIMARY_OFFLINE_TOKEN_ENV, null);
     public static final String PRIMARY_USERNAME = getOrDefault(PRIMARY_USERNAME_ENV, null);
-    public static final String PRIMARY_PASSWORD = getOrDefault(PRIMARY_PASSWORD_ENV, null);
 
     // sso.redhat.com secondary user (See README.md)
+    public static final String SECONDARY_OFFLINE_TOKEN = getOrDefault(SECONDARY_OFFLINE_TOKEN_ENV, null);
     public static final String SECONDARY_USERNAME = getOrDefault(SECONDARY_USERNAME_ENV, null);
-    public static final String SECONDARY_PASSWORD = getOrDefault(SECONDARY_PASSWORD_ENV, null);
 
     // sso.redhat.com alien user (See README.md)
-    public static final String ALIEN_USERNAME = getOrDefault(ALIEN_USERNAME_ENV, null);
-    public static final String ALIEN_PASSWORD = getOrDefault(ALIEN_PASSWORD_ENV, null);
+    public static final String ALIEN_OFFLINE_TOKEN = getOrDefault(ALIEN_OFFLINE_TOKEN_ENV, null);
 
     // sso.redhat.com different organization user with 1 SKU quota only for stage environment (See README.md)
-    public static final String DIFF_ORG_USERNAME = getOrDefault(DIFF_ORG_USERNAME_ENV, null);
-    public static final String DIFF_ORG_PASSWORD = getOrDefault(DIFF_ORG_PASSWORD_ENV, null);
+    public static final String DIFF_ORG_OFFLINE_TOKEN = getOrDefault(DIFF_ORG_OFFLINE_TOKEN_ENV, null);
 
+    public static final String ADMIN_OFFLINE_TOKEN = getOrDefault(ADMIN_OFFLINE_TOKEN_ENV, null);
```

Comment:
replace with 

which would later be naturally needed to be assigned accordingly to public Environment.<something> variables: 

```
    public static final String STRATOSPHERE_SCENARIO_1_USER_OFFLINE_TOKEN = getOrDefault(STRATOSPHERE_SCENARIO_1_USER_OFFLINE_TOKEN_ENV, null);
    public static final String STRATOSPHERE_SCENARIO_2_USER_OFFLINE_TOKEN = getOrDefault(STRATOSPHERE_SCENARIO_2_USER_OFFLINE_TOKEN_ENV, null);
    public static final String STRATOSPHERE_SCENARIO_3_USER_OFFLINE_TOKEN = getOrDefault(STRATOSPHERE_SCENARIO_3_USER_OFFLINE_TOKEN_ENV, null);
    public static final String STRATOSPHERE_SCENARIO_4_USER_OFFLINE_TOKEN = getOrDefault(STRATOSPHERE_SCENARIO_4_USER_OFFLINE_TOKEN_ENV, null);

```

Reply:
resolved